docs/semgrep-supply-chain/transitive-reachability.md

```diff
@@ -0,0 +1,116 @@
+---
+slug: transitive-reachability
+title: Transitive reachability
+hide_title: true
+description: Learn how transitive reachability identifies vulnerabilities introduced by third-party packages.
+tags:
+ - Semgrep Supply Chain
+---
+
+# Transitive reachability (beta)
+
+Semgrep Supply Chains' transitive reachability analysis helps you identify vulnerabilities introduced by third-party packages and their dependencies. 
```

Comment:
```suggestion
Semgrep Supply Chain's transitive reachability analysis helps you identify vulnerabilities introduced by third-party packages and their dependencies. 
```
sidenote, I think "identify" is weird here. We were already identifying them, we just didn't classify them very well.